packages/gatsby-plugin-utils/README.md

```diff
@@ -6,10 +6,38 @@
 npm install gatsby-plugin-utils
 ```
 
-### Example
+### validateOptionsSchema
+
+#### example
 
 ```js
 import { validateOptionsSchema } from "gatsby-plugin-utils"
 
 await validateOptionsSchema(pluginName, pluginSchema, pluginOptions)
 ```
+
+### testPluginOptionsSchema
+
+Utility to validate and test plugin options schemas. An example of a plugin options schema implementation can be found in the [gatsby-node.js file of gatsby-plugin-google-analytics](../gatsby-plugin-google-analytics/gatsby-node.js).
+
+#### example
+
+```js
```

Comment:
Might note that this is cypress testing

Reply:
Added a comment on it to mention the test runner 😊

Reply:
> Is the testPluginOptionsSchema code already somewhere else? 

Sorry, I missed the part of this: yes it is => https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-plugin-utils/src/test-plugin-options-schema.ts